roles/remove-node/pre-remove/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+- name: remove-node | Drain node except daemonsets resource
+  command: >-
+    {{ bin_dir }}/kubectl drain
+      --force
+      --ignore-daemonsets
+      --grace-period {{ drain_grace_period }}
+      --timeout {{ drain_timeout }}
+      --delete-local-data {{ item }}
+  with_items:
+    - "{{ groups['kube-node'] }}"
+  delegate_to: "{{ groups['kube-master'][0] }}"
+  ignore_errors: yes
```

Comment:
We should attempt to stop kubelet service if that host is still existing. It can have `failed_when: false` as part of the command, but the node will just re-register itself after deletion if kubelet is still running.